### PR TITLE
Cf active token for transaction

### PR DESCRIPTION
**Pull request by CFDevelopment**
- saves active token from the main page for when the user navigates to coin send

### CODE REVIEW

File: src/screens/main/portfolio/index.js [hunk 1]
```diff
@@ -67,26 +68,25 @@ class Portfolio extends Component {
...
   /**
-   * tokens are passed into the function where their symbols and addresses are parsed out and stored in an array, 
+   * tokens are passed into the function where their symbols and addresses are parsed out and stored in an array,
    * which is then passed to processAllTokenBalances. The return is the concatenated string of symbols in wallet, and the amount
    * of each token the user has for a given private key.
    */
   formatTokens = async (tokenList) => {
     let tokenObjectList = [];
-    let privateKey; 
+    let privateKey;
```
Comment (GregTheGreek) [inline src/screens/main/portfolio/index.js]: never used.

File: src/screens/main/tokens/send/CoinSend.js [hunk 2]
```diff
@@ -105,7 +102,7 @@ class CoinSend extends Component {
     if(response.flag) {
       this.setState({maliciousCheck: true});
     } else {
-      const amountString = `${  this.state.value  }`;
+      const amountString = `${this.state.value}`;
```
Comment (GregTheGreek) [inline src/screens/main/tokens/send/CoinSend.js]: const amountString = this.state.value.toString()

File: src/screens/main/portfolio/index.js (hunk 1 above)
Comment (CFDevelopment) [reply src/screens/main/portfolio/index.js]: When I look at this branch in active branches the code is being used.

formatTokens = async (tokenList) => {
    let tokenObjectList = [];
    let privateKey;
    for (let i = 0; i < tokenList.length; i++) {
      let tokenObj = {};
      tokenObj.symbol = tokenList[i].symbol;
      tokenObj.contractAddress = tokenList[i].address;
      tokenObjectList.push(tokenObj);
    }
    privateKey =  this.state.currentWallet.privateKey;
    return { tokenSymbolString, tokenBalances } = await processAllTokenBalances(privateKey, tokenObjectList);
  }

File: src/screens/main/tokens/send/CoinSend.js (hunk 2 above)
Comment (CFDevelopment) [reply src/screens/main/tokens/send/CoinSend.js]: @GregTheGreek fixed

File: src/screens/main/portfolio/index.js (hunk 1 above)
Comment (GregTheGreek) [reply src/screens/main/portfolio/index.js]: What i meant is let statement is redundant. Just use const below.

Comment (CFDevelopment) [reply src/screens/main/portfolio/index.js]: @GregTheGreek fixed

**Pull request merged**